src/debug/StaticWebAppDebugProvider.ts

```diff
@@ -63,6 +64,9 @@ export class StaticWebAppDebugProvider implements DebugConfigurationProvider {
                 }
 
                 if ((await tryGetApiLocations(context, folder))?.length) {
```

Comment:
Unrelated, but `tryGetApiLocations` can take a second or two (sometimes even double digits) to run.  If this is having every time the user tries to debug, I suggest that you look to see if they have an API path in the `swa.cli.config` rather than running `tryGetApiLocations`. 

The only case we should do that is if we were going to set the API path for them (but I assume that you only do that on the `swa.cli.config` init

Reply:
Ok, that's a very reasonable concern. 

Right now I invoke it every time, but I think what I should do is check to see if they have a swa-cli.config.json and if it has the apiLocation property (just like you mentioned).

And I'll only invoke it if they don't have a swa-cli.config.json file at all.

Currently I'm always invoking it because I want to help users add the apiLocation property to their swa-cli.config.json file if they don't already have it set up, but I think it's ok to remove that feature for sake of performance.